nextcloudappstore/core/views.py

```diff
@@ -40,10 +41,30 @@ class AppReleasesView(DetailView):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['categories'] = Category.objects.all()
+
+        releases = self.object.releases_by_platform_v()
+        nightlies = self.object.releases_by_platform_v(nightlies=True)
+        versions = set(list(releases.keys()) + list(nightlies.keys()))
```

Comment:
curious if this would work by using versions = set(zip(release.keys(), nightlies.keys())